docs/pywbemclifeatures.rst

```diff
@@ -289,20 +289,44 @@ The following are examples of scalar property definitions:
 Interactively selecting INSTANCENAME
 ------------------------------------
 
-The INSTANCENAME argument has a certain complexity, particularly for
-associations and for classes with multiple keys.
+The INSTANCENAME argument can be complex, particularly for associations and
+classes with multiple keys.
+
+To simplify using INSTANCENAME on the command line, pywbemcli provides a wild
+card character "?" that can be used with commands in place of the INSTANCENAME
```

Comment:
Suggest replacing "INSTANCENAME" with "instance name"

Reply:
How about the INSTANCENAME argument since that is what it is universally defined as.

DONE

Reply:
I'm on with "INSTANCENAME argument".